cmd/testdata/default.yaml

```diff
@@ -10,6 +10,9 @@ server:
   writeTimeout: 10s
   keyfile: "keyfile"
   certfile: "certfile"
+  disabledciphers:
+    - 0xc00a
+    - 0xc011
```

Comment:
Is it inconvenient for the user to make them figure out the hex value? Should it be a semantic string?

Reply:
Given other configuration formats, I'd expect people would expect string format.

Here is an example from a white-list configuration [sauce](https://jenda.hrach.eu/f2/cipherlist.html)
> ECDHE-RSA-AES256-GCM-SHA512:DHE-RSA-AES256-GCM-SHA512:ECDHE-RSA-AES256-GCM-SHA384:DHE-RSA-AES256-GCM-SHA384:ECDHE-RSA-AES256-SHA384